frontend/src/tests/components/Commons/CommonsList.test.js

```diff
@@ -89,4 +89,8 @@ describe("CommonsList tests", () => {
             i++;
         })
     });
+    test('displays default message when there are no commons', () => {
```

Comment:
This is correct!   I wonder if the only changes that should be in this PR are changes to this file and to `CommonsList.js`?